specification/redis/resource-manager/Microsoft.Cache/preview/2019-07-01/redis.json

```diff
@@ -1796,6 +1796,11 @@
           "type": "integer",
           "format": "int32",
           "description": "If clustering is enabled, the Shard ID of Redis Instance"
+        },
+        "isMaster": {
+          "readOnly": true,
+          "type": "boolean",
+          "description": "Specifies whether the instance is a master node."
```

Comment:
I know ARM guideline is to recommend we avoid booleans in the response, for 'isMaster', but I personally think its pretty close to future proof. There is a possible question of what will happen if we fail to determine which port is master... will the whole request fail?